how-to/how-to-template-guide.md

```diff
@@ -0,0 +1,149 @@
+- A list of favourites prepared, so you can manage them. For more information about favourites lists, see http://example.com/favorite_lists.
+```
+
+For easy understanding, consider classifying the prerequisites into different categories such as software prerequisites, and hardware prerequisites.
+
+Optionally, provide any helpful cues that signal to a user that they’re probably not in the right place and provide them with more suitable options. For example, If you are a Linux user, then refer to {link to relevant Linux how-to guide}.
+
+## About the "Steps" section
+
+The steps section is where you describe what the user needs to do. Use an ordered list structure to document the how-to steps. How you write your steps will vary depending on your organization's style guide. The template organizes the steps in the following way:
+
+{Task name}
+
+1. {Write the action to take here. Start with a verb.}
+
+   {Optional: Explanatory text}
+
+   {Optional: Code sample or screenshot that helps your users complete this step}
+
+   {Optional: Result}
+
+### Tips for writing steps
+
+- For task names, start with a bare infinitive also known as plain form or base form verbs. For example, “connect”, “set up”, or “build” and express the heading as a complete thought. Don't use the -ing form of the verb because it is harder to translate. Instead of saying, "Connect", you might say, "Connect to the VM instance”.
+- For each step, optionally provide some background information about the task so users know what they're about to do and why. Continuing with the example, you might provide some best practices for creating memorable repository names.
+- Optionally, add a code sample or screenshot after the explanatory text, depending on the type of how-to you're writing. Screenshots are a great way to show specific parts of the screen you are referring to in a step. Make sure your code samples work.
+- Remember to orient your users when walking them through each step. If they need to open a particular file or dialogue box to complete the task, provide that information first.
+- Provide examples of sample output such as return data, and a message so that the users can validate whether they performed the step correctly or not. Continuing with the example, you might describe what happens after a user clicks the "Create repository" button.
+- Use plain language and define the terminology of any technical term next to it.
+- Include one action in a step.
+
+Here is an example step:
+
+Create repository
+
+1. Enter a name for your new repository.
+   Good repository names are short and self-explanatory. Avoid repository names with three or more words. After you click "Create repository", GitHub creates your repository and the main page for the repository is displayed. If needed, you can add substeps below a primary step. Make sure to indent the substep one tab space over if you're using Markdown.
+
+Create pull request
+
+1. 2. To create a pull request, do the following:
```

Comment:
issue: Check how this is rendered. It comes out as 1. ii. where I expect we were going for 2.